src/assets/scripts/client/parsers/aircraftCommandParser/aircraftCommandDefinitions.js

```diff
@@ -75,7 +75,7 @@ const ZERO_ARG_AIRCRAFT_COMMANDS = {
         validate: zeroArgumentsValidator,
         parse: noop
     },
-    climbViaSID: {
+    climbViaSid: {
```

Comment:
👍 

I've never been a fan of the `climbViaSID` capitalization.  thank you for changing this!

Reply:
yeah, it used to make me _crazy_ to see `Sid` because that's like seeing `Usa`: it's just wrong. But I've come around to the convention of keeping _EVERYTHING_ proper camel cased. Makes it easy and consistent, so here we are. 😃